packages/flutter_tools/lib/src/windows/visual_studio_solution_utils.dart

```diff
@@ -0,0 +1,318 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../base/common.dart';
+import '../base/file_system.dart';
+import '../convert.dart';
+import '../globals.dart' as globals;
+import '../plugins.dart';
+import '../project.dart';
+import 'visual_studio_project.dart';
+
+// Constants corresponding to specific reference types in a solution file.
+// These values are defined by the .sln format.
+const String _solutionTypeGuidFolder = '2150E333-8FDC-42A3-9474-1A3956D46DE8';
```

Comment:
nit: use the `k` prefix for constant values

Reply:
Done.